dtfs-central-api/src/v1/routes/middleware/rateLimit/index.test.js

```diff
@@ -27,7 +27,7 @@ describe('createRateLimit', () => {
   };
 
   const invalidThresholdThrownError = 'Invalid rate limit threshold value.';
-  const invalidThresholdErrorMessageArgs = (threshold) => ['Invalid rate limit threshold value %s.', threshold];
+  const invalidThresholdErrorMessageArgs = (threshold) => ['Invalid rate limit threshold value %O.', threshold];
```

Comment:
```suggestion
  const invalidThresholdErrorMessageArgs = (threshold) => ['Invalid rate limit threshold value %i.', threshold];
```

Reply:
As mentioned - we should check were logging as expected

Reply:
updated to `%d`